mlflow/tracking/_tracking_service/utils.py

```diff
@@ -66,6 +64,7 @@ def set_tracking_uri(uri: Union[str, Path]) -> None:
 
         Current tracking uri: file:///tmp/my_tracking
     """
+    # pylint: enable=line-too-long
```

Comment:
Why do we need this?

Reply:
The bullets in the above docstring exceeded line length. I disabled pylint prior to docstring and reenable after. 

Wrapping the bullets solves the issue. Thanks for the spot!